raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -1,769 +1,541 @@
 
-Offences relating to pollution 
 (187) [1] Any person who: [a] discharges any dangerous materials, substances, oil, oil mixtures into land, water, air, or aquatic environment contrary to the provisions of this Act; [b] pollutes the environment contrary to the provisions of this Act; or [c] discharges any pollutant into the environment contrary to the provisions of this Act, commits an offence and is liable on conviction to a fine of not less than three million shillings but not exceeding fifty million shillings or to imprisonment for a term not exceeding twelve years, or to both. [2] In addition to any sentence that the court may impose upon a person convicted pursuant to the provisions of subsection [1], the court may direct that person to: [a] pay the full cost of cleaning up the polluted environment and of removing the pollution; or [b] clean up the polluted environment and remove the effects of pollution to the satisfaction of the Council. [3] Without prejudice to the provisions of subsections [1] and [2], the court may direct the polluter to meet the cost of injury or effect of the pollution to any third parties through adequate compensation, restoration or restitution.
 
-Offences relating to biological diversity 
-(188) A person who: [a] trades; [b] possesses; or [c] disturbs the habitat, of a component of biological diversity in contravention of guidelines and measures prescribed under sections 66, 67 and 68 or other provisions of this Act, commits, an offence and shall be liable on conviction to a fine not exceeding ten million shillings or to imprisonment for a term not exceeding five years or to both.
+(188) A person who: [a] trades; [b] possesses; or [c] disturbs the habitat, of a component of biological diversity in contravention of guidelines and measures prescribed under sections 66, 67 and 68 or other provisions of this Act, commits an offence and shall be liable on conviction to a fine not exceeding ten million shillings or to imprisonment for a term not exceeding five years or to both.
 
-Offences relating to environmental restoration, easement and conservation orders 
-(189) Any person who: [a] fails, neglects or refuses to comply with an environmental restoration order made under this Act; [b] fails, neglects or refuses to comply with an environmental easement, issued under this Act; [c] fails, neglects or refuses to comply with an environmental conservation order made under this Act, commits an offence and shall on conviction be liable to a fine not exceeding ten million shillings or to imprisonment for a term not exceeding five years, or to both. 
+(189) Any person who: [a] fails, neglects or refuses to comply with an environmental restoration order made under this Act; [b] fails, neglects or refuses to comply with an environmental e
```

Comment:
`one hundred thousand shilling` still needs to be replaced with `one hundred thousand shillings`.